brainrender/actors/streamlines.py

```diff
@@ -0,0 +1,154 @@
+import pandas as pd
```

Comment:
I would restructure this module to keep here (under `brainrender.actors`) the part that deals with the graphics, and move to the `atlas_specific` (but see relative note) the database part

Reply:
fair enough, I've done so.